### DISCUSSION TITLE
Cycle through windows including floating ones?

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5071] [Q&A]
I've been having a constant problem whenever I use Steam.
I'll open up one of the dozens of context windows that pop up and accidentally click the main steam window. Once this happens, the context window's focus is lost, it continues to draw on top of steam, and there is no way to refocus it unless I move the main steam window to another workspace.

I try using the MOD+homerow navigation keys, but they don't actually interact at all with floating windows.

**Comment by Airblader** [comment]
Context menus are "override redirect", meaning that they're not managed by i3 at all (which is why you cannot move them or tile them or anything like that). 

This sounds like a bug in steam.

**Reply by ghost** [reply]
There may have been a misunderstanding here, these aren't context menus similar to what you'd get by right clicking in a webpage, these are entire separate windows that can be moved independently of the parent window, like the history window in Firefox, just floating.

**Reply by Airblader** [reply]
Are you looking for `focus mode_toggle` then? In the default config this is bound to Mod1+space. I think I might not be understanding the situation correctly, though.
